src/templates/actors/parts/actor-spellbook.hbs

```diff
@@ -51,6 +51,9 @@
             <a class="item-control item-create" title="{{localize "SFRPG.SpellBook.CreateSpell"}}" {{#each section.dataset as |v k|}}data-{{k}}="{{v}}"{{/each}}>
                 <i class="fas fa-plus"></i> {{localize "SFRPG.SpellBook.Add"}}
             </a>
+            <a class="item-control item-browser" title="{{localize "SFRPG.SpellBook.CreateFromCompendium"}}" {{#each section.dataset as |v k|}}data-{{k}}="{{v}}"{{/each}} data-classes="{{json section.classes}}">
```

Comment:
Would also still like classes to just be their name, not a whole JSON

Reply:
done.